components/staging/staging.js

```diff
@@ -292,8 +293,11 @@ StagingViewModel.prototype.onAltEnter = function(d, e){
     }
     return true;
 };
+StagingViewModel.prototype.toggleWordWrap = function(state) {
+  this.wordWrap(state);
```

Comment:
I would either change the function to do this `this.wordWrap(!this.wordWrap());` or rename it to `wordWrapChange` (like `textDiffTypeChange`)